### DISCUSSION TITLE
Is there a right place to ask about possible extensions to junitxml?

### DISCUSSION THREAD

**Discussion by couling** [junit-team/junit-framework#3604] [Q&A]
junit's xml output has, to some extent taken on a life of it's own.  It get's used by other tools outside of the java world as a standard for communicating between test suit and other CI tools and reporting engines.  There are times other test tools might have information to convey that is not related to junit or even java.

I wanted to ask if the junit team take any responsibility for junitxml as a standard in its own right? Is there a better community to ask such questions?

----

_Simple example: [pytest cannot find somewhere to put python warnings as they're not stdout](https://github.com/pytest-dev/pytest/issues/2717).  In java these would be compile time warnings and so not visible to junit._

**Comment by marcphilipp** [comment]
We have started work on a new format that is extensible by design in https://github.com/ota4j-team/open-test-reporting.

**Reply by couling** [reply]
Good to know. Presumably this would require putting in feature requests to all the respective tool which have used junit's XML to ask them to implement open-test-reporting.  At first glance there's they're not compatible.  At this time (2023-12-15) open-test-reporting V1 hasn't been released.
